themes/learn.jquery.com/style.css

```diff
@@ -53,11 +53,11 @@ a {
 	top: 11px;
 	width: 120px;
 	background-color: #ffea94;
-	background-image: -webkit-gradient(linear, left top, left bottom, from(#ffea94), to(#FFEA5E)); 
-	background-image: -webkit-linear-gradient(top, #ffea94, #FFEA5E); 
-	background-image:    -moz-linear-gradient(top, #ffea94, #FFEA5E); 
-	background-image:     -ms-linear-gradient(top, #ffea94, #FFEA5E); 
-	background-image:      -o-linear-gradient(top, #ffea94, #FFEA5E); 
+	background-image: -webkit-gradient(linear, left top, left bottom, from(#ffea94), to(#FFEA5E));
+	background-image: -webkit-linear-gradient(top, #ffea94, #FFEA5E);
+	background-image:    -moz-linear-gradient(top, #ffea94, #FFEA5E);
+	background-image:     -ms-linear-gradient(top, #ffea94, #FFEA5E);
+	background-image:      -o-linear-gradient(top, #ffea94, #FFEA5E);
```

Comment:
True. But that's not something for this PR ;-) The only reason this appears in the diff is because the removed trailing whitespace.